src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -25,6 +25,16 @@ public CommandContext(String namespace, String collection) {
     this(namespace, collection, CollectionSettings.empty(), null, null, null);
   }
 
+  public static CommandContext from(
```

Comment:
java doc

Reply:
Added in this commit [b403d2c](https://github.com/stargate/data-api/pull/905/commits/b403d2c798a6226f0738fa42e582b7329b49f304).